site/docs/v1/tech/apis/webauthn.adoc

```diff
@@ -135,4 +144,140 @@ include::docs/v1/tech/apis/_import-webauthn-request-body.adoc[]
 
 Only a status code is available on the Import API, no JSON body will be returned.
 
-include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
\ No newline at end of file
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+
+== Start a WebAuthn Passkey Registration
+
+This API is used to start a WebAuthn registration ceremony by providing some details about the current user and the new passkey. The response contains the creation options that will be passed to the authenticator on the client device, including a one-time challenge unique to the current registration ceremony.
```

Comment:
```suggestion
This API is used to start a WebAuthn registration ceremony by providing some details about the current user and the new passkey. The response is a JSON object which is suitable to be passed to the WebAuthn JavaScript API and includes a one-time challenge unique to the current registration ceremony.
```

Any chance we can do the arraybuffer transforms in our output? That would be helpful for folks doing an integration (call the FusionAuth API, then plug the results directly into WebAuthn).

Reply:
It needs to be base64url-encoded to avoid encoding issues in-flight. I'm not sure how we would send/receive `ArrayBuffer`s on APIs.

I will provide sample JavaScript for doing the transformation. It's just a few lines each way.

Reply:
Perfect.